governance.md

```diff
@@ -0,0 +1,164 @@
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Op1st GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a subproject | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry|
+| Subproject owner | Set direction and priorities for a subproject | Demonstrated responsibility and excellent technical judgement for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
+
+### Community groups
+The Op1st community is initiating Governance 1.0 with only one type of community group, Special Interest Groups (SIGs).
+The purpose of doing so is to empower SIGs to gather interested people around a subject area, then decide and implement that into governance.
+For example, one of the first acts of the Community SIG is to establish a milestone schedule for defining and holding elections for future steering committee(s).
+Governance 1.1 will therefore include details on new community groups (subprojects and working groups).
+
+#### SIGs
+Special Interest Groups (SIGs) are persistent open groups that focus on a part of the project. SIGs must have open and transparent proceedings.
+Anyone is welcome to participate and contribute provided they follow the Op1st Code of Conduct.
+The purpose of a SIG is to own and develop a set of subprojects.
+
+A SIG can have its own policy for contribution, described in a README or CONTRIBUTING file in the SIG folder in this repo (e.g. sig-community/CONTRIBUTING.md), and its own mailing list, chat channel, etc.
```

Comment:
Is there any part of the project that is not in a SIG? If so, what differentiates the community work from SIG work? Lets answer this question and related stuff here.

Reply:
> Is there any part of the project that is not in a SIG? If so, what differentiates the community work from SIG work? Lets answer this question and related stuff here.

This seems like it's answered in the 1.1 future groups that follow? In that, the idea there is work outside of a SIG and its subprojects is addressed (committees, working groups) is included without yet prescribing the details and points of difference.

My reason for leaving the answer to your question until the 1.1 work is these are the kinds of things that seem writ in stone if they come in the first governance—instead of being ideas needing generating, it becomes existing policy that needs to be debated and changed. Yet the original debate to create the policy hasn't taken place! So then we're going back to interpret past intentions without the discussion record, etc.

What I think we can do best is capture all of these very good discussion points as starting points for the Community SIG work on Governance 1.1.

Reply:
Makes sense. The description below in 1.1 covers this.